dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -49,7 +48,7 @@ public class ZonedIntervalIterableTest {
   private static final ZonedParser zonedParserStart = ZonedParser.withDefaultPattern(START_OF_DAY);
   private static final ZonedParser zonedParserEnd = ZonedParser.withDefaultPattern(END_OF_DAY);
 
-  private void testIterable(int expectCount, @Nonnegative ZonedIntervalIterable iterable) {
+  private void testIterable(int expectCount, ZonedIntervalIterable iterable) {
```

Comment:
Most probably the intention was to use `@Nonnull` instead of `@Nonnegative`.

Reply:
It could be the case, but it's fine for the test to make `iterable` null - it's an invalid result from generator and it makes the test fail.